### PR TITLE
Fixed default value in the field Статуc

### PR DESCRIPTION
**Pull request by Illya97**
Follow the link: https://github.com/ita-social-projects/OoS-Frontend/issues/1331

### CODE REVIEW

File: src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <mat-form-field floatLabel="never" appearance="none">
     <mat-select disableOptionCentering panelClass="dropdown-panel" class="step-input"
       formControlName="institutionStatusId">
-      <mat-option [value]="0" class="dropdown-option"> Відсутній </mat-option>
+      <mat-option [value]="0" class="dropdown-option"> Працює </mat-option>
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.html:9]: revert this change. we push additional status to have such option ad "відсутній", but with this change, we will have two "працює".  Працює we receive from the back-end and can display it, and add additional status Відсутній.  The lost of options should be remained the same
image

File: src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts [hunk 2]
```diff
@@ -52,6 +54,17 @@ export class CreatePhotoFormComponent implements OnInit {
...
+    this.institutionStatuses$.pipe(takeUntil(this.destroy$)).subscribe((institutionStatus: InstitutionStatus[]) => {
+      this.institutionStatus = institutionStatus;
+      for (let i = 0; i < this.institutionStatus.length; i++) {
+        if (this.institutionStatus[i].name == "Працює") {
+          this.institutionStatus.splice(i,1);
+          break;
+        }
+      }
+    });
+
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts:67]: remove these changes. you should change the default one only 
image

Comment (litvinets) [reply src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts:67]: this is the only change that is needed

Comment (litvinets) [reply src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts:67]: put the id of Працює status

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by LeraKornienko — APPROVED** [review]

**Comment by litvinets** [comment]
it doesn't work. On update the status is sending as null, there should be an id 
image

File: src/app/shared/constants/constants.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ export class Constants {
 
   static readonly SCROLL_TO_TOP_BUTTON_POS = 300;
   static readonly SOCIAL_GROUP_ID_ABSENT_VALUE = 0;
-  static readonly INSTITUTION_STATUS_ID_ABSENT_VALUE = 0;
+  static readonly INSTITUTION_STATUS_ID_ABSENT_VALUE = 1;
```
[the remark this answers is not in the document]
Comment (litvinets) [reply src/app/shared/constants/constants.ts:28]: revrt this change, thr absent value should be remained as null

**Review by litvinets — CHANGES_REQUESTED** [review]
needs work

**Review by litvinets — CHANGES_REQUESTED** [review]
Freezed until the back-end is ready

**Review by sivxkxn — APPROVED** [review]
Wrote a comment related class of Constants, but the PR is good.

File: src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts [hunk 4]
```diff
@@ -52,6 +55,12 @@ export class CreatePhotoFormComponent implements OnInit {
     this.store.dispatch(new GetInstitutionStatus());
     this.provider ? this.activateEditMode() : this.onAddForm();
     this.passPhotoFormGroup.emit(this.PhotoFormGroup);
+    this.institutionStatuses$.pipe(takeUntil(this.destroy$),filter((institutionStatuses: InstitutionStatus[])=>(!!institutionStatuses))).subscribe((institutionStatuses: InstitutionStatus[]) => {
```
Comment (sivxkxn) [inline src/app/shell/personal-cabinet/provider/create-provider/create-photo-form/create-photo-form.component.ts:58]: formmating

**Review by litvinets — APPROVED** [review]

**Comment by Illya97** [comment]
Fixed

**Pull request merged**